classes/Domain/Talk/TalkFilter.php

```diff
@@ -71,25 +71,18 @@ public function getFilteredTalks(int $adminUserId, $filter = null)
         switch (\strtolower($filter)) {
             case 'selected':
                 return $this->talk->selected();
-
             case 'notviewed':
                 return $this->talk->notViewedBy($adminUserId);
-
             case 'notrated':
                 return $this->talk->notRatedBy($adminUserId);
-
             case 'toprated':
                 return $this->talk->topRated();
-
             case 'plusone':
                 return $this->talk->ratedPlusOneBy($adminUserId);
-
             case 'viewed':
                 return $this->talk->viewedBy($adminUserId);
-
             case 'favorited':
                 return $this->talk->favoritedBy($adminUserId);
-
```

Comment:
I kinda like blank lines before `case`/`default`, I must say. 😕

Reply:
I don't mind removing the configuration for `case` statements, as I personally never use `switch` statements.

Reply:
This case is actually handled by configuring the `return` token: it removes extra consecutive blank lines after a `return` statement.